goto_commands.py

```diff
@@ -0,0 +1,136 @@
+import sublime
+import sublime_plugin
+
+from itertools import dropwhile, takewhile
+from .lint import persist
+
+
+"""
+Implement typical Goto Next Previous Error Commands. If you will, add
+the following to your key bindings.
+
+```
+  { "keys": ["ctrl+k", "n"],
+    "command": "sublime_linter_goto_error",
+    "args": {"direction": "next"}
+  },
+  { "keys": ["ctrl+k", "p"],
+    "command": "sublime_linter_goto_error",
+    "args": {"direction": "previous"}
+  },
+```
+
+Supported args:
+
+* `direction`: 'previous' | 'next'
+* `count`: How wide you will jump. Defaults to 1.
+* `wrap`: Set to True, to jump to the top if you're on the last error and
```

Comment:
Perhaps we need to flash a message on the status bar that says we wrapped?

Reply:
I don't think so, bc we already have the visual feedback. So the user has the immediate feedback that the command has been received. 

With wrap `false`, I flash bc the user cannot decide if the computer actually received the command bc nothing happens otherwise.

Reply:
Sounds good, agreed.